Makefile

```
@@ -0,0 +1,28 @@
+.PHONY: test lint build tidy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)